pandas/core/frame.py

```diff
@@ -6093,7 +6119,7 @@ def _arith_op(left, right):
         new_data = self._dispatch_frame_op(other, _arith_op)
         return new_data
 
-    def _construct_result(self, result) -> DataFrame:
+    def _construct_result(self, result: DataFrame) -> DataFrame:
```

Comment:
this is not complete, this would be an ndarray, scalar or DataFrame

Reply:
essentially anything that can be input to the constructor. leave it out for now.

Reply:
Done